util/uvmdvgen/env_pkg.sv.tpl

```diff
@@ -16,6 +16,7 @@ package ${name}_env_pkg;
   import cip_base_pkg::*;
 % endif
 % if has_ral:
```

Comment:
Perhaps also need to add `lowrisc:dv:dv_base_reg` to `env.core.tpl`.

Reply:
hm I think this would only matter if an  IP had a RAL model but for some reason was not a comportable IP (since the `cip_lib.core` has `dv_base_reg` as a dependency) - is that a case we should cover?